lib/json_api_resource/handlers/find_handler.rb

```diff
@@ -0,0 +1,19 @@
+module JsonApiResource
+  module Handlers
+    class FindHandler
+      attr_accessor :result
+
+      def initialize(results)
+        simplify!(results)
+      end
+
+      def simplify!(results)
+        result = results.first
+
+        result.meta = results.meta
+
+        result.linked_data = results.try(:linked_data)
+      end
```

Comment:
It's turning the `[Client]` into `Client` it seems. `find` has that weird behavior that doesn't conform to any of the other functions